src/app/pcasts/dao/users_dao.py

```diff
@@ -66,3 +66,23 @@ def search_users(search_name, offset, max_search):
       (User.username.like(search_name+'%')) \
       .offset(offset).limit(max_search).all()
   return possible_users
+
+def change_user_name(old_name, new_name):
```

Comment:
This function would allow any user to change any other users' name.  We should not allow this.  This should accept a `user_id`, which would be the `id` of the user making the request.